nni/retiarii/evaluator/functional.py

```diff
@@ -26,6 +26,7 @@ def _load(ir):
 
     def _dump(self):
         return {
+            'type_name': self.__class__,
```

Comment:
what is the type of `self.__class__`? is it safe to dump this `self.__class__` object and load it in another environment (e.g., another machine)?

Reply:
Its type is "type". `nni.dump` and `nni.load` will handle the serialization of a "type" object.

1. If we can get its path, we dump it into a path. e.g., `torch.nn.Conv2d`.
2. Otherwise, we dump it into a binary. e.g., class defined in a jupyter notebook.

Reply:
Since it's a class object, why the key is called "type name"?

Reply:
Good question. I copied from `operation.py`. Let me fix it.